aws/awsresource.go

```diff
@@ -27,13 +27,11 @@ const (
 
 // basic AWS resource, has properties that most/all resources have
 type AWSResource struct {
-	ID          reapable.ID
-	Name        string
-	Region      reapable.Region
-	AWSState    AWSState
-	Description string
-	VPCID       string
-	OwnerID     string
+	ID                 reapable.ID
```

Comment:
Clean up some of the fields of AWSResource that aren't universal across resources